tests/e2e/assets/webpack/test-server-app/app/injectable.ts

```diff
@@ -4,5 +4,5 @@ import {DOCUMENT} from '@angular/platform-browser';
 
 @Injectable()
 export class MyInjectable {
-  constructor(public viewContainer: ViewContainerRef, @Inject(DOCUMENT) public doc) {}
```

Comment:
VCR is not an ordinary injectable

It can only be injected from a component and it's value is the VCR of the specific instance of the component.

A service/ injectable is not able to inject a VCR